spec/factories/availability_zone.rb

```diff
@@ -3,6 +3,8 @@
     sequence(:name)     { |n| "availability_zone_#{seq_padded_for_sorting(n)}" }
   end
 
+  factory :availability_zone_generic, :parent => :availability_zone
```

Comment:
@syncrou The tests pass locally if I just use `availability_zone` in the tests.  Is this needed for some reason?

Reply:
@gmcculloug - I'll do more testing - I was seeing failures without the `:parent` value in place.